data/transaction.go

```diff
@@ -191,8 +191,15 @@ type ResponseMultipleTransactions struct {
 
 // TxCostResponseData follows the format of the data field of a transaction cost request
 type TxCostResponseData struct {
-	TxCost     uint64 `json:"txGasUnits"`
-	RetMessage string `json:"returnMessage"`
+	TxCost     uint64                                     `json:"txGasUnits"`
+	RetMessage string                                     `json:"returnMessage"`
+	ScResults  map[string]*ApiSmartContractResultExtended `json:"smartContractResults"`
+}
+
+// ApiSmartContractResultExtended extends the structure transaction.ApiSmartContractResult with an extra field
+type ApiSmartContractResultExtended struct {
```

Comment:
rename to `ExtendedApiSmartContractResult`

Reply:
done.